src/consts.rs

```diff
@@ -90,12 +90,12 @@ impl Constant {
 impl PartialEq for Constant {
     fn eq(&self, other: &Constant) -> bool {
         match (self, other) {
-            (&Constant::Str(ref ls, ref lsty), &Constant::Str(ref rs, ref rsty)) => ls == rs && lsty == rsty,
+            (&Constant::Str(ref ls, ref l_sty), &Constant::Str(ref rs, ref r_sty)) => ls == rs && l_sty == r_sty,
```

Comment:
I don’t think there is any value added by all those `_`, IMO it’s as easy to distinguish between `l` and `r` as between `l_` and `r_`. The lint seems more annoying than usefull :s